ompi/mca/pml/ucx/pml_ucx.c

```diff
@@ -958,7 +958,7 @@ int mca_pml_ucx_send(const void *buf, size_t count, ompi_datatype_t *datatype, i
     return mca_pml_ucx_send_nb(ep, buf, count, datatype,
                                mca_pml_ucx_get_datatype(datatype),
                                PML_UCX_MAKE_SEND_TAG(tag, comm), mode,
-                               mca_pml_ucx_send_completion);
```

Comment:
you can remove this argument, because this func is called from one place only. Better specify empty callback inside
`mca_pml_ucx_send_nb` directly